addons/website/views/website_navbar_templates.xml

```diff
@@ -97,43 +97,64 @@
                                             <p>New Page</p>
                                         </a>
                                     </div>
-                                    <div groups="base.group_system" name="module_website_blog" t-att-data-module-id="env.ref('base.module_website_blog').id" t-att-data-module-shortdesc="env.ref('base.module_website_blog').shortdesc" class="col-md-4 mb8 o_new_content_element">
+                                    <div groups="base.group_system" name="module_website_blog"
+                                        t-att-data-module-id="env['ir.model.data'].xmlid_to_res_id('base.module_website_blog')"
+                                        t-att-data-module-shortdesc="env.ref('base.module_website_blog').shortdesc"
```

Comment:
But you are still doing a `ref` call?

Reply:
The other one cannot be avoided without a new method in ir.model.data ^^.
At least we gain one ref ^^